eox_nelp/signals/tests/test_receivers.py

```diff
@@ -481,3 +486,58 @@ def test_event_is_emitted(self, tracker_mock):
                 "created": block.created,
             }
         )
+
+
+class IncludeTrackerContextTestCase(unittest.TestCase):
+    """Test class for include_tracker_context method."""
+
+    def test_context_is_included(self):
+        """
+        This tests that the body kwargs has been updated after the method execution.
+
+        Expected behavior:
+            - body kwargs is equal to the tracker context
+        """
+        body = {"kwargs": {}}  # This is the default value for kwargs
+        context = {"This is a fake context": True}
+
+        # Set tracker context
+        tracker = get_tracker()
```

Comment:
Could you say me where is registered the tracker?
https://github.com/openedx/event-tracking/blob/master/eventtracking/tracker.py#L134

Reply:
witchcraft 
https://github.com/openedx/event-tracking/blob/master/eventtracking/django/django_tracker.py#L172

Reply:
omg https://github.com/openedx/event-tracking/blob/master/eventtracking/django/apps.py#L23 
Dark magicians xD